### DISCUSSION TITLE
Technology Stack

### DISCUSSION THREAD

**Discussion by sandy-marrero** [uprm-inso4117-2023-2024-s2/semester-project-uprmarketplace#1] [Ideas]
Hey everyone, let's use this disccusion to go over the possible choices for our frontend and backend for the project. Please let us know your suggestions.

**Comment by sandy-marrero** [comment]
Frontend - React Native

**Comment by YeranConcepcion** [comment]
FrontEnd-Angular + Bootstrap + Material

Backend- Serverless AWS

**Comment by BreannaGSegarra** [comment]
Frontend - Astro
Backend - Python (FastAPI)
Database - PostgreSQL

**Comment by Chosen-Juan1** [comment]
Backend- Python (Flask)
Database - PostgreSQL

**Reply by GaddPetrivoki** [reply]
This with Frontend React Native seems like the most straightforward
